packages/react-hooks/src/useMappedSelection.ts

```diff
@@ -25,6 +25,7 @@ export function useMappedSelection<TItem, TValue>(
     );
 
     return {
+      // TODO: Need to handle this... not sure how yet
```

Comment:
Not sure what this means? Can you add any more details to the comment?

Reply:
Whoops, I had already handled it. Basically handled it by throwing if an `undefined` is returned from the `getItem` call.